### ISSUE TITLE
How to build and run?

### ISSUE THREAD

**Issue by ifredom** [DavidQinqianWu/YYM_Type#2]
How to build and run?

**Comment by DavidQinqianWu** [comment]
yes, OSMD is too heavy that dont really kind for mobile app , especially, use osmd to render the complicated score . so i give up with osmd
